package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "npm explore @daostack/arc -- npm install openzeppelin-solidity @daostack/infra --no-save && npm explore @daostack/infra -- npm install openzeppelin-solidity --no-save && npm explore @daostack/arc-hive -- npm install openzeppelin-solidity --no-save",
+    "verify.build": "node node_modules/typescript/bin/tsc verify.ts -sourceMap",
+    "verify.help": "node verify.js -h",
+    "verify.lint": "tslint -c tslint.json verify.ts",
+    "verify.lint.andFix": "npm run verify.lint -- --fix",
+    "verify": "node verify.js"
```

Comment:
can be replaced with `ts-node` as I mentioed (take a look at [this](https://github.com/TypeStrong/ts-node/issues/313) for a solution to a problem you might encounter)

Reply:
On your suggestion, I looked into using ts-node, but it seems there are a lot of issues with it and using source maps in debuggers.  Since the current soluton works for now, I'm not inclined to going down that rabbit hole.